src/main/java/io/vertx/ext/amqp/impl/ReplyManager.java

```diff
@@ -32,10 +32,9 @@
 public class ReplyManager {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(ReplyManager.class.getName());
-  public static final String REPLY_TO_MESSAGE_PROPERTY = "reply-to-message";
+  public static final String REPLY_TO_MESSAGE_PROPERTY = "correlation_id";
```

Comment:
Not actually an issue any longer given that the implicit reply functionality has been removed (yay), but for clarity and later reference this isn't what I was talking about with my original review comment.

This simply altered the name of the 'proprietary' application property being used. It still wasn't doing what the bridge was doing, which was to use the built in message-id and correlation-id fields of the AMQP message Properties section and not defining a custom entry in ApplicationProperties which is distinct.